src/vsync.js

```diff
@@ -51,12 +71,36 @@ export class Vsync {
    * @param {!Object<string, *>|undefined} opt_state
    */
   run(task, opt_state) {
+    // Do not request animation frames when the document is not visible.
+    if (!viewerFor(this.win).isVisible()) {
```

Comment:
viewer.isVisible is false when docState.isVisible or the iframe is hidden, right?

I think that is what we want.